newsletter-january-2023.rst

```diff
@@ -0,0 +1,134 @@
+  You can watch it here: `Documenting Django Code in 2022`_
+- 🛠️ In line with the talk,
+  we continued reorganizing our own documentation to follow the `Diátaxis Framework`_.
+- 🚢️ Sphinx 6.0 and 6.1 have been released.
+  If you want things to not break,
+  it's good to wait a bit.
+  Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- 🔒️ We fixed a security vulnerability on our platform. See: `GHSA-368m-86q9-m99w`_
+- 🔒 We immediately rotated Circle CI secrets as `advised by Circle CI`_.
+  Read the Docs did not share secrets with Circle CI for build infrastructure,
+  nor do we see any signs that other secrets have been suspiciously accessed.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _Documenting Django Code in 2022: https://www.youtube.com/watch?v=mqn0D4xat58
+.. _Diátaxis Framework: https://diataxis.fr/
+.. _GHSA-368m-86q9-m99w: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-368m-86q9-m99w
+.. _advised by Circle CI: https://circleci.com/blog/january-4-2023-security-alert/
+
+Upcoming features
+-----------------
+
+..
+  Notes:
+
+  Next newsletter:
+  Make a general announcement of our Roadmap
+
+  General:
+
+  When creating newsletter drafts, we keep the items here from the previous newsletter.
+  This is in order to ensure due follow-up on features that are announced publicly.
+  
+  Feature done? A great follow-up is to add what was previously an upcoming feature as a released feature in the former section.
+  
+  Feature not done?
+  Make sure that upcoming features are announced with a link to issues or PRs where the progress can be seen.
+  If this is done, then subsequent newsletters aren't compelled to share progress when it's uninteresting.
+  
+  If a feature was announced as upcoming but isn't yet released,
+  then try rephrasing the announcement as a general news update about the progress and where it can be followed.
+
+The core team has laid out its 2023 Q1 roadmap and our general focus will be on:
+
+- Improving support for using *any* documentation tool in the build process.
+- Currently, users can either customize a default build process or overwrite the entire process from scratch.
+  We are aligning these two approaches,
+  making it easier to design and configure builds for a project's individual needs.
+- Continuing to restructure and rewrite the user documentation of Read the Docs.
+  We look forward to publishing the restructured documentation which is likely happening at the end of Q1.
+- Dashboard redesign: Our admin dashboard is currently being redesigned and reimplemented.
+  We are hoping to have a public beta in Q1.
+
+Interested in all the details? `View our full Roadmap 📍️`_
+
+.. _View our full Roadmap 📍️: https://github.com/orgs/readthedocs/projects/156/views/4
```

Comment:
@ericholscher pay attention to this one! @humitos and I was on a call and he suggested that we add a button with the roadmap since we are talking a lot about it. We then looked at a previous mention of the roadmap, where it was said that we would announce it in the future.

We agreed to do a soft announcement and then perhaps write something more in the next newsletter, so we can brew up at text that tells readers what the intention is with this link -- like why should you bother, how do you tell us that you want a bug fix or a feature etc.

Reply:
Yea, makes sense 👍